paper/paper.md

```diff
@@ -1,351 +1,412 @@
+(2) Give a clear overview of what user can expect from playing with existing examples,
+which are aimed at reproducing literature examples
+
+### Collisional Breakup
+`PySDM` was recently used as a calibration tool to generate data for learning microphysics rate
+  parameters in @Bieli_et_al_2022 (in review). 
+Particles in a box environment undergo coalescence and breakup with a fixed coalescence 
+  efficiency, and the moments of the distribution are used as training data. 
+In addition, two figures from a forthcoming publication @DeJong_et_al_2022 that describes the
+  physics and algorithm for superdroplet breakup are included. 
+The first example (reproduced in \autoref{fig:readme_fig_1}), demonstrates the impact of including
+  the breakup process on the particle size distribution, versus a coalescence-only case. 
+The second similarly demonstrates the impact of the breakup process in a one-dimensional setup 
+  based on the example of @Shipway_and_Hill_2012.
+
+\begin{figure}[h]
+    \centering
+    \includegraphics[width=0.6\textwidth]{deJong_fig1}
+    \caption{Sample plot generated with the code snippets included in the paper.}
+    \label{fig:dJ_fig_1}
+\end{figure}
+
+### Immersion freezing 
+This release of PySDM introduces representation of immersion freezing, 
+  i.e. freezing contingent on the presence of insoluble ice nuclei immersed 
+  in supercooled water droplets.
+There are two alternative models implemented, in both cases the formulation
+  is probabilistic and based on Poissonian model of heterogeneous freezing.
+The two models embrace, so-called, singular and time-dependent approaches and
+  are based on the formulation presented in @Shima_et_al_2020 and
+  @Alpert_and_Knopf_2016, respectively.
+In the singular model, the relevant introduced particle attribute is the freezing temperature
+  which is randomly sampled at initialisation from an ice nucleation active sites (INAS) model;
+  subsequently freezing occurs in a deterministic way upon encountering ambient 
+  temperature that is lower than the particle's freezing temperature.
+In the time-dependent model, the relevant introduced particle attribute is the insoluble
+  material surface which is randomly sampled at initialisation; 
+  freezing is triggered by evaluating probability of freezing at instantaneous
+  ambient conditions and comparing it with a random number.
+For the time-dependent model, the water Activity Based Immersion Freezing Model (ABIFM)
+  of @Knopf_and_Alpert_2013 is used.
+  
+For validation of the the newly introduced immersion freezing models, a set of
+  notebooks reproducing box-model simulations from @Alpert_and_Knopf_2016 was introduced
+  to the PySDM-examples package.
+A comparison of the time-dependent and singular models using the kinematic
+  prescribed-flow environment introduced in PySDM v1 has been developed
+  and is the focus of @Arabas_et_al_2022.
+
+### Surface-partitioning of organics to modify surface tension of droplets
```

Comment:
@slayoo This "example" is not yet merged into main. It's something that will be part of the paper Ryan and I working on, but that paper won't be ready in time for this JOSS submission. Should I try to merge a first version of this example that just has the Köhler curves now?

Reply:
I think so. We have four such cases here: breakup, adaptivity, freezing & surfactants. In all cases, PySDM-examples would include drafts of examples at the time of JOSS submission. I'd argue that doing it this way allows us to cite the JOSS paper in the respective non-software papers, which is the key point of having the JOSS paper out, right?

Reply:
Yes, sounds good. I will open that PR to examples soon :)

Reply:
PR here: https://github.com/atmos-cloud-sim-uj/PySDM-examples/pull/152